streams/src/test/java/org/apache/kafka/streams/state/internals/SessionBytesStoreTest.java

```diff
@@ -68,18 +65,18 @@
 
 public abstract class SessionBytesStoreTest {
 
-    protected static final long SEGMENT_INTERVAL = 60_000L;
-    protected static final long RETENTION_PERIOD = 10_000L;
+    static final long SEGMENT_INTERVAL = 60_000L;
+    static final long RETENTION_PERIOD = 10_000L;
 
-    protected SessionStore<String, Long> sessionStore;
-    protected InternalMockProcessorContext context;
-
-    private final List<KeyValue<byte[], byte[]>> changeLog = new ArrayList<>();
+    SessionStore<String, Long> sessionStore;
 
+    private InternalMockProcessorContext context;
     private final Producer<byte[], byte[]> producer = new MockProducer<>(true,
         Serdes.ByteArray().serializer(),
         Serdes.ByteArray().serializer());
 
+    private final MockRecordCollector recordCollector = new MockRecordCollector();
```

Comment:
Can we just set the `recordCollector` in `setUp`, instead of keeping a persistent one? It just makes it easier to see that the state really will get cleaned in between tests.

Reply:
Ack.